sidebars.js

```diff
@@ -91,6 +91,12 @@ module.exports = {
                   type: "doc",
                   id: "tools/chrome-extension/viewing-insights-and-invitations",
                 },
+                {
+                  type: "file",
+                  label: "feature 7",
```

Comment:
I do not feel this is the best label for features but just went as the previous error, we can get all the features in one page in another PR.